### ISSUE TITLE
Fluent Use

### ISSUE THREAD

**Issue by Wigmund** [microsoft/microsoft-ui-xaml#2792]
Should controls automatically use the fluent design language in WinUI 3.0 preview 1 or is there something i should enable application wide? All controls seems flat, with no fluent elements?

**Comment by shaheedmalik** [comment]
I think they should default to fluent. With the option to disable it. This would promote fluent usage.

**Comment by mdtauk** [comment]
Some types of Acrylic are not working currently in WinUI 3 - so things like the NavigationView don't have them set as default.

Controls like Menus and Flyouts will use InApp Acrylic as expected, by default.

**Comment by StephenLPeters** [comment]
Yes, they should. They don't right now, and this is a known issue.

**Comment by Wigmund** [comment]
The XAML system is great for customizing UI appearance, its one of its strengths, but I wish they provided a standard "Windows10VersionWhatever" style for all controls that can be used by applications (and is used by all inbox apps) to make life more consistent. Then when a new design language is necessary for future versions of windows a new "Windows10NextVersion" style is made available for use for everything - would go along way to unifying inbox apps and making life easier for developers who want their apps to blend in to the latest windows appearance.

**Comment by ranjeshj** [comment]
@chigy as FYI. The default styles will be updated to the newest design language to update the appearance. We do have some known issues in WinUI3 as @mdtauk  mentioned earlier. There are no plans to ship version specific styles.
